src/index.ts

```diff
@@ -49,8 +51,8 @@ function parseXPath(xpathString: string) {
 }
 
 /**
- *
- * @param xpathString The XPath for which a buckets hould be retrieved
+ * @public
+ * @param xpathString - The XPath for which a buckets hould be retrieved
```

Comment:
Could you also fix the sentence? should be `The XPath for which buckets should be retrieved`